main.yml

```diff
@@ -435,8 +435,32 @@ subscriptions:
   frontend:
     paths:
       - /staging/subscriptions
+    sub_apps:
+      - id: subscriptions-rhel
   git_repo: https://github.com/RedHatInsights/curiosity-frontend
 
+subscriptions-rhel:
+  title: Red Hat Enterprise Linux
+  frontend:
+    sub_apps:
+      - id: all
+        title: All
+        reload: rhel/all
+        default: true
+      - id: arm
+        title: ARM
+        reload: rhel/arm
+      - id: ibmpower
+        title: IBM Power
+        reload: rhel/ibmpower
+      - id: ibmz
+        title: IBM Z systems
+        reload: rhel/ibmz
+      - id: x86
+        title: x86
+        reload: rhel/x86
+    suppress_id: true
```

Comment:
I made this field to create a kind of exception for the "settings" sidenav item, since it wasn't an existing app that we still wanted to show. I'd like to avoid using this if possible, but I guess it depends on what you're trying to do with these sub-apps

Reply:
Going back through and looking at removing `supress_id` along with @karelhala recommendation of avoiding `reload`